packages/scene-composer/package.json

```diff
@@ -132,8 +132,8 @@
   "dependencies": {
     "@aws-sdk/client-iottwinmaker": "3.335.0",
     "@awsui/collection-hooks": "^1.0.51",
-    "@awsui/components-react": "^3.0.0",
-    "@awsui/design-tokens": "^3.0.41",
+    "@awsui/components-react": "^3.0.900",
+    "@awsui/design-tokens": "^3.0.50",
```

Comment:
This package was intentionally downgraded in a previous [PR](https://github.com/awslabs/iot-app-kit/pull/1722/files). Please confirm that this change doesn't introduce any breaks in storybook or console (both dev 
& production mode).

Reply:
Testing has been validated.